packages/luigis-box/src/Model/Product/Parameter/Value/ParameterValueRepository.php

```diff
@@ -36,4 +36,25 @@ public function getExistingParameterValuesByValuesAndLocale(array $parameterValu
             ])
             ->orderBy(OrderByCollationHelper::createOrderByForLocale('pv.text', $locale))->getQuery()->getResult();
     }
+
+    /**
+     * @param string[] $parameterValues,
+     * @param string $locale
+     * @return array
+     */
+    public function getSliderParameterValuesForMinAndMaxByLocale(
+        array $parameterValues,
+        string $locale,
+    ): array {
+        return $this->em->createQueryBuilder()
+            ->select('pv')
+            ->from(ParameterValue::class, 'pv')
+            ->where('pv.numericValue IN(:parameterValues)')
```

Comment:
should not here be `BETWEEN`? I mean, this way you will get two `ParameterValue` instances that match the minimal and maximal value from the input `$parameterValues` array, right? I thought you needed to get all the values within the input range :thinking: If I am right, maybe it would be worth adding also some check that the input array really contains exactly two items

Reply:
Actually, for slider you need only two, because of ProductFilterOption::__construct

Reply:
ok, I have to believe you :smile: